sdk/anomalydetector/azure-ai-anomalydetector/tests/test_anomaly_detector.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+# coding=utf-8
+
+import functools
+import pytest
+import time
+import os
+from datetime import datetime
+
+from devtools_testutils import (
+    AzureRecordedTestCase,
+    PowerShellPreparer,
+    recorded_by_proxy,
+    EnvironmentVariableLoader,
+)
+from azure.ai.anomalydetector import AnomalyDetectorClient
+from azure.ai.anomalydetector.models import *
+from azure.core.credentials import AzureKeyCredential
+
+from test_data import get_test_data
+
+
+class TestAnomalyDetector(AzureRecordedTestCase):
+    @recorded_by_proxy
```

Comment:
`@SchemaRegistryPreparer` is lacked. Please follow [guidance](https://github.com/Azure/azure-sdk-for-python/wiki/Dataplane-Codegen-Quick-Start-for-Test#writing-new-tests) to write test with standard test framework which is convenient to add more test in the future.
You can also refer to other service: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/loadtestservice/azure-developer-loadtesting/tests/test_smoke.py